.circleci/config.yml

```diff
@@ -30,7 +30,7 @@ jobs:
          command: make test
   e2e:
     machine:
-      image: circleci/classic:latest
+      image: ubuntu-1604:201903-01
```

Comment:
We Ubuntu now? Not that I object, but why the change?

Reply:
It's not strictly needed.
`circleci/classic:latest` is Ubuntu 14.04 which has a very old version of `make`.

I wanted to use a `file` function (introduced in make 4.1) which was later made capable of *reading a file* (introduced in make 4.2)

ubuntu-1604 has make 4.1 not 4.2 :-(

But I think it's worth upgrading to a supported version anyway and this is now the recommended machine image:
  https://circleci.com/docs/2.0/executor-types/#using-machine

classic remains the default for backwards compatibility reasons 
  https://discuss.circleci.com/t/default-machine-executor-image-update/29308

Reply:
Huh, today I learnt that `make` is getting new features.

In any case I'm happy to use the CircleCI recommended machine image. (Ubuntu 14.04 is pretty old).